<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>825</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>15000</width>
    <height>15000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Редактор расписания</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0.933, y1:0.926136, x2:0, y2:0, stop:0.0571429 rgba(255, 255, 255, 255), stop:1 rgba(150, 205, 83, 202));</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" rowspan="2">
    <widget class="QListWidget" name="FunctionsLW">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>410</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>800</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <property name="text">
       <string>Редактор расписания</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset resource="ImageResourses.qrc">
        <normaloff>:/images/Edit.png</normaloff>:/images/Edit.png</iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Редактор справочников</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset resource="ImageResourses.qrc">
        <normaloff>:/images/dictionary.png</normaloff>:/images/dictionary.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Настройки</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset resource="ImageResourses.qrc">
        <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Справка</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset resource="ImageResourses.qrc">
        <normaloff>:/images/information.png</normaloff>:/images/information.png</iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Выйти</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset resource="ImageResourses.qrc">
        <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QLabel" name="Description_l">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15000</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="MainInscription_l">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>807</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="text">
      <string>Редактор расписания</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="RunDialogBt">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ImageResourses.qrc">
       <normaloff>:/images/begin.png</normaloff>:/images/begin.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Functions_l">
     <property name="font">
      <font>
       <family>URW Bookman L</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="text">
      <string>Выберите один изпунктов меню:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImageResourses.qrc"/>
 </resources>
 <connections/>
</ui>
